packages/security_center/test/test_utils.dart

```diff
@@ -70,14 +84,47 @@ AppPermissionsService registerMockAppPermissionsService({
   return service;
 }
 
-LocalSnapData registerMockLocalSnapData({
-  List<Snap> snaps = const [],
-}) {
+LocalSnapData registerMockLocalSnapData({List<Snap> snaps = const []}) {
```

Comment:
some more formatting changes (they all seem to be related to trailing commas :confused: )